google/resource_google_project.go

```diff
@@ -429,7 +432,12 @@ func updateProjectBillingAccount(d *schema.ResourceData, config *Config) error {
 	if name != "" {
 		ba.BillingAccountName = "billingAccounts/" + name
 	}
-	_, err := config.clientBilling.Projects.UpdateBillingInfo(prefixedProject(pid), ba).Do()
+	updateBillingInfoFunc := func() error {
+		var err error
```

Comment:
nit: this is equivalent to `_, err := config.clientBilling.Projects.UpdateBillingInfo(prefixedProject(pid), ba).Do()`, right?

Reply:
Yes, I think it is the same. I think that there is some worry about the value for 'err' persisting past the function closure (if err was actually declared before the function), but I am not even sure if that would happen if there was an 'err' defined before this function.

Will change.